packages/office-ui-fabric-react/src/components/pickers/TagPicker/TagItem.scss

```diff
@@ -2,15 +2,18 @@
 
 .root {
   @include focus-border();
-  flex-shrink: 0;
+  flex-shrink: 1;
   background: $ms-color-neutralLighter;
   height: 32px;
   line-height: 32px;
   cursor: default;
   user-select: none;
   display: flex;
   flex-wrap: nowrap;
+  min-width: 100px;
   max-width: 100%;
+  margin: 2px 2px;
```

Comment:
Sharing the screen shots. 
**Img with original padding 1px** 

![captureoriginal](https://cloud.githubusercontent.com/assets/25333246/24882921/b04698a4-1df7-11e7-8d24-b6cde791c691.PNG)

**Img with only padding=4px**

![capturepadding4px](https://cloud.githubusercontent.com/assets/25333246/24882945/c4332e18-1df7-11e7-9296-2f50040385c3.PNG)

**Img with padding and margin:**

![capturepaddingandmargin](https://cloud.githubusercontent.com/assets/25333246/24882958/d17fef02-1df7-11e7-8da9-a4be3eeae59b.PNG)